R/duration.R

```diff
@@ -0,0 +1,62 @@
+#' A duration class for mdates
+#'
+#' @description
+#' The `mdates_duration` class introduces methods that annotate a duration or
+#' period with representations of its uncertainty.
```

Comment:
I think this needs more detail. Perhaps explaining it in comparison to how durations are handled in lubridate might be helpful?

Reply:
I have added more details to documentation, however, lubridate makes a distinction between duration (exact time) and periods (aproximation for dates) - see https://cran.r-project.org/web/packages/lubridate/lubridate.pdf. For now, perhaps re-think these if/when we add times to messydates? Please let me know what you think. Thank you.

Reply:
Thanks @henriquesposito . We can make a note to review this again later.